gapis/gfxapi/gfxapi.proto

```diff
@@ -50,6 +50,9 @@ enum ResourceType {
 	ShaderResource = 5;
 	// ProgramResource represents the Program resource type
 	ProgramResource = 6;
+	// UninitializedTextureResource represents texture resource type
+	// which has been generated but does not have dimensions yet.
+	UninitializedTextureResource = 7;
```

Comment:
I am aware of that - however, we seem to tie the dimensions for the whole trace and the "the dimensions were not assigned" should be one of them.